docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,626 @@ license: |
+    </tr>        
+    <tr>
+      <td> <b>{percentile_approx | percentile_approx}</b>(<i>e: Column, percentage [, frequency]</i>)</td>
+      <td>Column name; percentage is a number between 0 and 1; frequency is a positive integer</td>
+      <td>Returns the approximate percentile value of numeric column at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td> <b>{percentile_approx | percentile_approx}</b>(<i>e: Column, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>Column name; percentage is a number between 0 and 1; frequency is a positive integer</td>
+      <td>Returns the approximate percentile value of numeric column at the given percentage.</td>
+    </tr>             
+    <tr>
+      <td> <b>{stddev_samp | stddev | std}</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the sample standard deviation calculated from values of a group</td>
+    </tr>  
+    <tr>
+      <td> <b>stddev_pop</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the population standard deviation calculated from values of a group</td>
+    </tr>
+    <tr>
+      <td> <b>{variance | var_samp}</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the sample variance calculated from values of a group</td>
+    </tr>    
+    <tr>
+      <td> <b>sum</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the sum calculated from values of a group.</td>
+    </tr>       
+    <tr>
+      <td> <b>var_pop</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the population variance calculated from values of a group</td>
+    </tr>        
+    <tr>
+      <td> <b>collect_list</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Collects and returns a list of non-unique elements. The function is non-deterministic because the order of collected results depends on the order of the rows which may be non-deterministic after a shuffle</td>
+    </tr>       
+    <tr>
+      <td> <b>collect_set</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Collects and returns a set of unique elements. The function is non-deterministic because the order of collected results depends on the order of the rows which may be non-deterministic after a shuffle.</td>
+    </tr>
+    <tr>
+        <td> <b>count_min_sketch</b>(<i>e: Column, eps: double, confidence: double, seed integer</i>)</td>
+        <td>Column name; eps is a value between 0.0 and 1.0; confidence is a value between 0.0 and 1.0; seed is a positive integer</td>
+        <td>Returns a count-min sketch of a column with the given esp, confidence and seed. The result is an array of bytes, which can be deserialized to a `CountMinSketch` before usage. Count-min sketch is a probabilistic data structure used for cardinality estimation using sub-linear space..</td>
+    </tr>
+  </tbody>
+</table>
+
+### Example
```

Comment:
Nit: ```Example``` -> ```Examples```?
Sometimes you have a blank line between examples, sometimes you don't. I guess make it consistent and always have a blank line in between?

Reply:
thanks, done